packages/dx-react-grid-demos/src/demo-sources/grid-detail-row/detail-row-controlled.jsxt

```diff
@@ -32,7 +37,10 @@ export default class Demo extends React.PureComponent {
         { name: 'city', title: 'City' },
         { name: 'car', title: 'Car' },
       ],
-      rows: generateRows({ length: 8 }),
+      rows: generateRows({
```

Comment:
We can extract a function (maybe 'generateUniqueRows') that will return rows with ids